src/Components/RestaurantContainer.jsx

```diff
@@ -18,16 +21,31 @@ export default function RestaurantContainer({ restaurantId }) {
   }, []);
 
   const restaurant = useSelector(get('restaurant'));
+  const accessToken = useSelector(get('accessToken'));
 
   if (!restaurant) {
     return (
       <p>Loading...</p>
     );
   }
 
+  function handleChange({ name, value }) {
+    dispatch(changeReviewField({ name, value }));
+  }
+
+  function handleSubmit() {
+    dispatch(sendReview({ restaurantId }));
+  }
+
   return (
     <>
       <RestaurantDetail restaurant={restaurant} />
+      {accessToken ? (
+        <ReviewForm
+          onChange={handleChange}
+          onSubmit={handleSubmit}
+        />
+      ) : null}
```

Comment:
JSX에서 `false` , `true` , `undefined` , `null` 은 아무것도 렌더하지 않습니다. 따라서 3항 연산자를 사용해서 null을 반환하는 대신에 그냥 false를 반환해도 될 것 같습니다.

See also
- [[https://reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored]()](https://reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored)
```suggestion
      {accessToken && (
        <ReviewForm
          onChange={handleChange}
          onSubmit={handleSubmit}
        />
      )}
```